react-frontend/src/queries/guide/guides.js

```diff
@@ -0,0 +1,15 @@
+import gql from 'graphql-tag';
+
+const GUIDES_QUERY = gql`
+  query Guides {
+    guides {
+      Title
+      Summary
+      UID
+      createdAt
+      updatedAt
+    }
```

Comment:
Why are some variables uppercase and some lowercase?

Reply:
fixed